package.json

```diff
@@ -16,6 +16,7 @@
         "@dhis2/d2-i18n-generate": "1.2.0",
         "@dhis2/d2-ui-core": "7.1.3",
         "@dhis2/d2-ui-forms": "^6.5.3",
+        "@dhis2/ui": "6.12.0",
         "@dhis2/ui-core": "^4.8.0",
         "@dhis2/ui-widgets": "^2.0.8",
```

Comment:
@tokland Can we delete the ui-widgets dependency? I believe it was only used for the header, right?

Reply:
Good point. Removed from the app, unfortunately still being used by 2 (one, us!):

```
$ yarn why "@dhis2/ui-widgets" | grep depends
   - "@dhis2#ui#@dhis2#ui-forms" depends on it
   - "@eyeseetea#d2-ui-components#@dhis2#ui#@dhis2#ui-forms" depends on it
```